website/docs/reference/widgets/datepicker.md

```diff
@@ -1,132 +1,168 @@
-| **onFocus** | Sets an action to take place when a datepicker is focused. Can be set from the GUI list of common actions ([examples here](../appsmith-framework/widget-actions/)), or you can define a custom JavaScript function to call instead. |
-| **onBlur** | Sets an action to take place when a datepicker loses focus. Can be set from the GUI list of common actions ([examples here](../appsmith-framework/widget-actions/)), or you can define a custom JavaScript function to call instead. |
+## Display date dynamically
 
-### Label
+You can display the date dynamically based on user interaction or by fetching the date from a query response.
 
-The property hosts a group of configurations that you can use to associate a display name and define a placement for the widget. These properties are usually useful when you want to design forms that follow a defined alignment for your form fields and give a professional look to your forms. Below are the properties that you can use:
 
-| Label         | Description                                                                                          |   |
-| ------------- | ---------------------------------------------------------------------------------------------------- | - |
-| **Text**      | Sets the label text of the widget.                                                                   |   |
-| **Position**  | Sets where the label appears relative to the widget's input area. Choose between Left, Top, or Auto. |   |
-| **Alignment** | Sets whether the label is left- or right-aligned.                                                    |   |
-| **Width**     | Sets the width of the label. The number represents how many characters/columns wide the label is.    |   |
+---
 
-| Label Styles         | Description                                                                                                                          |   |
-| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------ | - |
-| **Text Color**       | Sets the text color for the label. Accepts valid CSS [`color` ](https://developer.mozilla.org/en-US/docs/Web/CSS/color)values.       |   |
-| **Text Size**        | Sets the size of the label font. Accepts valid CSS [`font-size`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-size) values. |   |
-| **Label Font Style** | Toggles font styles (**bold** or _italic)._                                                                                          |   |
+**Example**: suppose you want to display the date of birth of users in the [mock database](https://docs.appsmith.com/core-concepts/connecting-to-data-sources/connecting-to-databases#sample-databases).
```

Comment:
write similar to the one on JSON form where we help the user understand the context better